aws/datadog_integration_role.yaml

```diff
@@ -32,6 +32,13 @@ Parameters:
       https://docs.datadoghq.com/integrations/amazon_cloudtrail/
     Type: String
     Default: ''
+  ResourceCollectionPermissions:
+    Type: String
+    Default: false
+    AllowedValues:
+      - true
+      - false
+    Description: Set this value to "true" if you wish to use the compliance integration in Datadog and we will install the necessary permissions.
```

Comment:
This can match the more generic resource collection description like 
https://github.com/DataDog/cloudformation-template/blob/50d7befd1ef3cfb4ea18809bf0cd9f556463da4b/aws/main.yaml#L76

Reply:
Thanks for the catch, updated.